Cargo.toml

```diff
@@ -15,6 +15,8 @@ nix = "0.17.0"
 libproc = "0.7.2"
 libc = "0.2.72"
 object = "0.20"
+capstone = "0.7.0"
```

Comment:
Just a general comment: one alternative to consider is [iced-x86](https://github.com/0xd4d/iced/tree/master/src/rust/iced-x86) which is written in pure Rust (though it lacks support for ARM and other ISAs at this time). It's not important at this stage, but I think in the longer term targeting 100% Rust code might be better.

Reply:
> iced-x86 is a high performance and **correct** x86 (16/32/64-bit) instruction decoder, disassembler and assembler written in Rust.

Being "correct" is a bold claim for an x86 disassembler. x86 is extremely complex.

Reply:
Yeah, I see that Capstone is also used by Radare2 and Frida, I was unaware of that.
A library with such a wide community support is definitely preferable!